src/pages/PublicModule/PublicModule.tsx

```diff
@@ -132,7 +132,7 @@ class PublicModule extends Component<PublicModuleProps, MyModuleState> {
           <Card.Body>
             <Card.Title>{module.name}</Card.Title>
             <Card.Text style={{height: 105, textOverflow: 'ellipsis', overflow: 'hidden'}}>
-              {module.description.substring(0, 150)}
+              {module.description}
```

Comment:
In the `renderStartButton` method, you need to link the user back to the login page and change the text to login instead of "start"